app/controllers/admin/api/registry/policies_controller.rb

```diff
@@ -32,13 +35,39 @@ def create
     respond_with current_account.policies.create(policy_params)
   end
 
+  ##~ e = sapi.apis.add
+  ##~ e.path = "/admin/api/registry/policies/{id}.json"
+  ##~ e.responseClass = "policy"
+  #
+  ##~ op             = e.operations.add
+  ##~ op.httpMethod  = "GET"
+  ##~ op.summary     = "APIcast Policy Registry Read"
+  ##~ op.description = "Returns the APIcast policy by ID"
+  ##~ op.group       = "apicast_policies"
+  #
+  ##~ op.parameters.add @parameter_access_token
+  ##~ op.parameters.add @parameter_policy_id
+  def show
+    respond_with(policy)
+  end
+
   private
 
+  attr_reader :policy
+
   def authorize_policies
     authorize! :manage, :policy_registry
   end
 
   def policy_params
     params.require(:policy).permit(:name, :version, :schema)
   end
+
+  def policy_id_param
+    params[:id].to_s.match(/([\w.-]+?)(?:\.(json|xml))?$/)[1]
```

Comment:
I think we should drop the requirement for finding by `name + version`
I do not remember well the use case but if we have a CRUD then it should not be difficult to use.